Racing/PlayerCar.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Racing
+{
+    public class PlayerCar
```

Comment:
Inheritance should have used for that purpose.
Class Car should inherit class Node.

Reply:
-2 to OOD.